decidim-meetings/app/cells/decidim/meetings/cancel_registration_meeting_button_cell.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Meetings
+    # This cell renders the button to cancel a meeting registation.
+    class CancelRegistrationMeetingButtonCell < Decidim::ViewModel
+      include Decidim::IconHelper
+      include MeetingCellsHelper
+
+      def show
+        return unless model.can_be_joined_by?(current_user)
+        return unless model.has_registration_for?(current_user)
+
+        render
+      end
+
+      private
+
+      delegate :current_user, to: :controller, prefix: false
+
+      def current_component
```

Comment:
The current user is already defined in `Decidim::ViewModel`
```suggestion
      private

      def current_component
```